.resinci.json

```diff
@@ -0,0 +1,47 @@
+{
+  "dependencies": {
+    "linux": [
+      "libudev-dev",
+      "libusb-1.0-0-dev",
+      "libyaml-dev"
+    ]
+  },
+  "builder": {
+    "appId": "io.resin.etcher",
+    "copyright": "Copyright 2016 Resinio Ltd",
+    "productName": "Etcher",
+    "mac": {
+      "category": "public.app-category.developer-tools"
+    },
+    "dmg": {
+      "iconSize": 110,
+      "contents": [
+        {
+          "x": 140,
+          "y": 225
+        },
+        {
+          "x": 415,
+          "y": 225,
+          "type": "link",
+          "path": "/Applications"
+        }
+      ],
+      "window": {
+        "width": 540,
+        "height": 405
+      }
+    },
+    "linux": {
+      "category": "Utility",
+      "packageCategory": "utils",
+      "synopsis": "Etcher is a powerful OS image flasher built with web technologies to ensure flashing an SDCard or USB drive is a pleasant and safe experience. It protects you from accidentally writing to your hard-drives, ensures every byte of data was written correctly and much more."
+    },
+    "deb": {
+      "priority": "optional",
+      "depends": [
```

Comment:
Why is the list of dependencies here so much shorter than the equivalent list at https://github.com/resin-io/etcher/blob/master/electron-builder.yml#L50 ?

Reply:
That big list contains dependencies for every Electron application. The Resin Concourse Electron pipeline keeps track of those, and requires projects to only declare the ones that are specific to the application.

Reply:
Ahhh, I didn't realise this was specifically a "Concourse Electron pipeline" and not just a "Concourse pipeline"! Perhaps it'd be worth specifying that in `.resinci.json` in case we add other non-Electron pipelines to Concourse in future?

Reply:
Yeah, I think that's a good point. I'll do it in a minute.

Reply:
Done!